### DISCUSSION TITLE
error: Package requirements (protobuf >= 2.6.0) were not met

### DISCUSSION THREAD

**Discussion by noloader** [protobuf-c/protobuf-c#665] [Q&A] [answered]
After a `git clone`:

```
$ cd protobuf-c
$ ./autogen.sh
...
$ ./configure
...
checking whether g++ supports C++11 features with -std=c++11... yes
checking for protobuf... no
checking for protobuf... no
configure: error: Package requirements (protobuf >= 2.6.0) were not met:

No package 'protobuf' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables protobuf_CFLAGS
and protobuf_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

-----

`libprotobuf-c-dev` is installed.

I don't see an `INSTALL` file with detailed instructions. The `README.md` does say:

> protobuf-c requires a C compiler, a C++ compiler, protobuf, and pkg-config to be installed.

And then execute the following:

```
./configure && make && make install
```

But running `configure` is produces the error above.

I found one similar issue at [Error when running ./configure](https://github.com/protobuf-c/protobuf-c/issues/189), but it did not lend insight into the issue.

-----

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 9.8 (stretch)
Release:        9.8
Codename:       stretch

$ gcc --version
gcc (Debian 6.3.0-18+deb9u1) 6.3.0 20170516
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

**Comment by edmonds** [comment, marked as answer]
`libprotobuf-c-dev` is the Debian package of the `protobuf-c` headers and static library. You can also install the `protobuf-c-compiler` package and skip building this repository.

If you actually do need to build this repository from source (e.g. because you need proto3 support which isn't available in the version of protobuf-c in stretch), then you need to install the build deps, i.e., run `apt build-dep protobuf-c`, or install the packages listed here: https://salsa.debian.org/edmonds/protobuf-c/blob/master/debian/control#L7-10.
